### DISCUSSION TITLE
获取最新评论，返回无效请求？

### DISCUSSION THREAD

**Discussion by jerryc127** [ArtalkJS/Artalk#415] [Q&A] [answered]
`http://xxxx/api/stat?type=latest_comments&limit=6&site_name=Butterfly`

调用网址没有错，但是返回无效请求
`{"success":false,"msg":"无效请求，Origin 无法获取","data":{}}`

**Comment by qwqcode** [comment]
考虑到安全性，API HTTP 请求需要传递 Origin 标头或 Referer 标头

文档说明：https://artalk.js.org/guide/backend/config.html#%E5%8F%AF%E4%BF%A1%E5%9F%9F%E5%90%8D-trusted-domains
相关代码：https://github.com/ArtalkJS/Artalk/blob/master/server/middleware/site_origin.go#L88

**Reply by jerryc127** [reply]
> 考虑到安全性，API HTTP 请求需要传递 Origin 标头或 Referer 标头
> 
> 文档说明：https://artalk.js.org/guide/backend/config.html#%E5%8F%AF%E4%BF%A1%E5%9F%9F%E5%90%8D-trusted-domains 相关代码：https://github.com/ArtalkJS/Artalk/blob/master/server/middleware/site_origin.go#L88

我已经加了 origin 还是不行
![image](https://user-images.githubusercontent.com/16351105/217455357-e092fbde-7114-4fee-93e6-6a599e24f1a7.png)

![image](https://user-images.githubusercontent.com/16351105/217454781-8e54e9b9-1d82-4616-879e-8e1cf9e524a8.png)

**Reply by jerryc127** [reply]
> 考虑到安全性，API HTTP 请求需要传递 Origin 标头或 Referer 标头
> 
> 文档说明：https://artalk.js.org/guide/backend/config.html#%E5%8F%AF%E4%BF%A1%E5%9F%9F%E5%90%8D-trusted-domains 相关代码：https://github.com/ArtalkJS/Artalk/blob/master/server/middleware/site_origin.go#L88

评论能加载
最新评论不能获取

**Reply by jerryc127** [reply, marked as answer]
> 考虑到安全性，API HTTP 请求需要传递 Origin 标头或 Referer 标头
> 
> 文档说明：https://artalk.js.org/guide/backend/config.html#%E5%8F%AF%E4%BF%A1%E5%9F%9F%E5%90%8D-trusted-domains 相关代码：https://github.com/ArtalkJS/Artalk/blob/master/server/middleware/site_origin.go#L88

无语了 捣鼓了那么就还是不能显示
结果把浏览器的广告插件关了
可以显示了。。
无语
